www/templates/build.html

```diff
@@ -23,36 +23,45 @@
 
             </div>
 
-            {% if not b.isFinished() %}
-            <div class="divider"></div>
-            {% if authz.advertiseAction('stopBuild', request) %}
-            <h2 class="head-2">
-                Stop Build
-            </h2>
-            {{ forms.stop_build(build_url+"/stop"+codebases_arg, authz, on_all=False, short=False, label='This Build')
-            }}
-            <div class="divider"></div>
-            {% endif %}
+            {% if b.isResuming() %}
+                <div class="divider"></div>
+                <h2 class="head-2">
+                        Stop Build
+                </h2>
+                    {{ forms.stop_build(build_url+"/cancel"+codebases_arg, authz, on_all=False, short=False, label='This Build')
```

Comment:
Maybe we should create a jinja macro for this passing in the url difference and the label?